File: Desktop/AsmaPravin/Masiudeen/Main_Transfer/Main_Transfer/eco_mfi/lib/pages/workflow/Savings/bean/SavingsListBean.dart
import 'package:eco_mfi/db/TablesColumnFile.dart';
class   SavingsListBean {
  int trefno;
  int mrefno;
  int msvngacctrefno;
  int msvngaccmrefno;
  int mlbrcode;
  int mcusttrefno;
  int mcustmrefno;
  String mprdacctid;
  String mlongname;
  String mmobno;
  String mprdcd;
  String mcurcd;
  DateTime mdateopen;
  DateTime mdateclosed;
  int mfreezetype;
  int macctstat;
  int mcustno;
  double macttotbalfcy;
  double mtotallienfcy;
  int mmoduletype;
  int mcenterid;
  int mgroupcd;
  DateTime mcreateddt;
  String mcreatedby;
  DateTime mlastupdatedt;
  String mlastupdateby;
  String mgeolocation;
  String mgeolatd;
  String mgeologd;
  DateTime mlastsynsdate;
  String mcrs;
  DateTime mcollectiondate;
  double mcollectedamount;
  String mremark;
  String mcashflow;
  DateTime mentrydate;
  String mbatchcd;
  String merrormessage;
  int msetno;
  int mscrollno;
  int missynctocoresys;
  int srno;
  DateTime moperationdate;
  SavingsListBean(
      {
        this.trefno,
        this.mrefno,
        this.msvngacctrefno,
        this.msvngaccmrefno,
        this.mlbrcode,
        this.mcusttrefno,
        this.mcustmrefno,
        this.mprdacctid,
        this.mlongname,
        this.mmobno,
        this.mprdcd,
        this.mcurcd,
        this.mdateopen,
        this.mdateclosed,
        this.mfreezetype,
        this.macctstat,
        this.mcustno,
        this.macttotbalfcy,
        this.mtotallienfcy,
        this.mmoduletype,
        this.mcenterid,
        this.mgroupcd,
        this.mcreateddt,
        this.mcreatedby,
        this.mlastupdatedt,
        this.mlastupdateby,
        this.mgeolocation,
        this.mgeolatd,
        this.mgeologd,
        this.mlastsynsdate,
        this.mcrs,
        this.mcollectiondate,
        this.mcollectedamount,
        this.mremark,
        this.mcashflow,
        this.mentrydate,
        this.mbatchcd,
        this.merrormessage,
        this.msetno,
        this.mscrollno,
      this.missynctocoresys,
      this.moperationdate});

  @override
  String toString() {
    return 'SavingsListBean{trefno: $trefno, mrefno: $mrefno, msvngacctrefno: $msvngacctrefno, msvngaccmrefno: $msvngaccmrefno, mlbrcode: $mlbrcode, mcusttrefno: $mcusttrefno, mcustmrefno: $mcustmrefno, mprdacctid: $mprdacctid, mlongname: $mlongname, mmobno: $mmobno, mprdcd: $mprdcd, mcurcd: $mcurcd, mdateopen: $mdateopen, mdateclosed: $mdateclosed, mfreezetype: $mfreezetype, macctstat: $macctstat, mcustno: $mcustno, macttotbalfcy: $macttotbalfcy, mtotallienfcy: $mtotallienfcy, mmoduletype: $mmoduletype, mcenterid: $mcenterid, mgroupcd: $mgroupcd, mcreateddt: $mcreateddt, mcreatedby: $mcreatedby, mlastupdatedt: $mlastupdatedt, mlastupdateby: $mlastupdateby, mgeolocation: $mgeolocation, mgeolatd: $mgeolatd, mgeologd: $mgeologd, mlastsynsdate: $mlastsynsdate, mcrs: $mcrs, mcollectiondate: $mcollectiondate, mcollectedamount: $mcollectedamount, mremark: $mremark, mcashflow: $mcashflow, mbatchcd: $mbatchcd, merrormessage: $merrormessage, msetno: $msetno, mscrollno: $mscrollno}';
  }

  factory SavingsListBean.fromMap(Map<String, dynamic> map) {
    //print(map);
    return SavingsListBean(
      trefno:map[TablesColumnFile.trefno] as int,
      mrefno:map[TablesColumnFile.mrefno] as int,
      msvngacctrefno:map[TablesColumnFile.msvngacctrefno] as int,
      msvngaccmrefno:map[TablesColumnFile.msvngaccmrefno] as int,
      mlbrcode:map[TablesColumnFile.mlbrcode] as int,
      mcusttrefno:map[TablesColumnFile.mcusttrefno] as int,
      mcustmrefno:map[TablesColumnFile.mcustmrefno] as int,
      mprdacctid:map[TablesColumnFile.mprdacctid] as String,
      mlongname:map[TablesColumnFile.mlongname] as String,
      mmobno:map[TablesColumnFile.mmobno] as String,
      mprdcd:map[TablesColumnFile.mprdcd] as String,
      mcurcd:map[TablesColumnFile.mcurcd] as String,
      mdateopen:(map[TablesColumnFile.mdateopen]=="null"||map[TablesColumnFile.mdateopen]==null)?null:DateTime.parse(map[TablesColumnFile.mdateopen]) as DateTime,
      mdateclosed:(map[TablesColumnFile.mdateclosed]=="null"||map[TablesColumnFile.mdateclosed]==null)?null:DateTime.parse(map[TablesColumnFile.mdateclosed]) as DateTime,
      mfreezetype:map[TablesColumnFile.mfreezetype] as int,
      macctstat:map[TablesColumnFile.macctstat] as int,
      mcustno:map[TablesColumnFile.mcustno] as int,
      macttotbalfcy:map[TablesColumnFile.mactTotbalfcy] as double,
      mtotallienfcy:map[TablesColumnFile.mtotallienfcy] as double,
      mmoduletype:map[TablesColumnFile.mmoduletype] as int,
      mcenterid:map[TablesColumnFile.mcenterid] as int,
      mgroupcd:map[TablesColumnFile.mgroupcd] as int,
      mcreateddt:(map[TablesColumnFile.mcreateddt]=="null"||map[TablesColumnFile.mcreateddt]==null)?null:DateTime.parse(map[TablesColumnFile.mcreateddt]) as DateTime,
      mcreatedby:map[TablesColumnFile.mcreatedby] as String,
      mlastupdatedt:(map[TablesColumnFile.mlastupdatedt]=="null"||map[TablesColumnFile.mlastupdatedt]==null)?null:DateTime.parse(map[TablesColumnFile.mlastupdatedt]) as DateTime,
      mlastupdateby:map[TablesColumnFile.mlastupdateby] as String,
      mgeolocation:map[TablesColumnFile.mgeolocation] as String,
      mgeolatd:map[TablesColumnFile.mgeolatd] as String,
      mgeologd:map[TablesColumnFile.mgeologd] as String,
      mlastsynsdate:(map[TablesColumnFile.mlastsynsdate]=="null"||map[TablesColumnFile.mlastsynsdate]==null)?null:DateTime.parse(map[TablesColumnFile.mlastsynsdate]) as DateTime,
      mcrs:map[TablesColumnFile.mcrs] as String,
      mcollectiondate:(map[TablesColumnFile.mcollectiondate]=="null"||map[TablesColumnFile.mcollectiondate]==null)?null:DateTime.parse(map[TablesColumnFile.mcollectiondate]) as DateTime,
      mcollectedamount:	map[TablesColumnFile.mcollectedamount] as double,
      mremark:map[TablesColumnFile.mremark] as String,
      mcashflow:map[TablesColumnFile.mcashflow] as String,
      mentrydate:map[TablesColumnFile.mentrydate]=="null" || map[TablesColumnFile.mentrydate]==null?null:DateTime.parse(map[TablesColumnFile.mentrydate]) as DateTime,
      mbatchcd:map[TablesColumnFile.mbatchcd] as String,
      merrormessage:map[TablesColumnFile.merrormessage] as String,
      msetno:map[TablesColumnFile.msetno] as int,
      mscrollno:map[TablesColumnFile.mscrollno] as int,
        missynctocoresys: map[TablesColumnFile.missynctocoresys] as int,
        moperationdate:map[TablesColumnFile.moperationdate] == "null" || map[TablesColumnFile.moperationdate] == null
        ? null
            : DateTime.parse(map[TablesColumnFile.moperationdate]) as DateTime,
    );
  }
  factory SavingsListBean.fromMapMiddleware(Map<String, dynamic> map,bool isFromMiddleware){
    print("fromMap");
    return SavingsListBean(
      trefno:map[TablesColumnFile.trefno] as int,
      mrefno:map[TablesColumnFile.mrefno] as int,
      mlbrcode:map[TablesColumnFile.mlbrcode] as int,
      mcusttrefno:map[TablesColumnFile.mcusttrefno] as int,
      mcustmrefno:map[TablesColumnFile.mcustmrefno] as int,
      mprdacctid:map[TablesColumnFile.mprdacctid] as String,
      mlongname:map[TablesColumnFile.mlongname] as String,
      mmobno:map[TablesColumnFile.mmobno] as String,
      mprdcd:map[TablesColumnFile.mprdcd] as String,
      mcurcd:map[TablesColumnFile.mcurcd] as String,
      mdateopen:(map[TablesColumnFile.mdateopen]=="null"||map[TablesColumnFile.mdateopen]==null)?null:DateTime.parse(map[TablesColumnFile.mdateopen]) as DateTime,
      mdateclosed:(map[TablesColumnFile.mdateclosed]=="null"||map[TablesColumnFile.mdateclosed]==null)?null:DateTime.parse(map[TablesColumnFile.mdateclosed]) as DateTime,
      mfreezetype:map[TablesColumnFile.mfreezetype] as int,
      macctstat:map[TablesColumnFile.macctstat] as int,
      mcustno:map[TablesColumnFile.mcustno] as int,
      macttotbalfcy:map[TablesColumnFile.mactTotbalfcy] as double,
      mtotallienfcy:map[TablesColumnFile.mtotallienfcy] as double,
      mmoduletype:map[TablesColumnFile.mmoduletype] as int,
      mcenterid:map[TablesColumnFile.mcenterid] as int,
      mgroupcd:map[TablesColumnFile.mgroupcd] as int,
      mcreateddt:(map[TablesColumnFile.mcreateddt]=="null"||map[TablesColumnFile.mcreateddt]==null)?null:DateTime.parse(map[TablesColumnFile.mcreateddt]) as DateTime,
      mcreatedby:map[TablesColumnFile.mcreatedby] as String,
      mlastupdatedt:(map[TablesColumnFile.mlastupdatedt]=="null"||map[TablesColumnFile.mlastupdatedt]==null)?null:DateTime.parse(map[TablesColumnFile.mlastupdatedt]) as DateTime,
      mlastupdateby:map[TablesColumnFile.mlastupdateby] as String,
      mgeolocation:map[TablesColumnFile.mgeolocation] as String,
      mgeolatd:map[TablesColumnFile.mgeolatd] as String,
      mgeologd:map[TablesColumnFile.mgeologd] as String,
      mlastsynsdate:(map[TablesColumnFile.mlastsynsdate]=="null"||map[TablesColumnFile.mlastsynsdate]==null)?null:DateTime.parse(map[TablesColumnFile.mlastsynsdate]) as DateTime,
      mcrs:map[TablesColumnFile.mcrs] as String,
      mcollectiondate:(map[TablesColumnFile.mcollectiondate]=="null"||map[TablesColumnFile.mcollectiondate]==null)?null:DateTime.parse(map[TablesColumnFile.mcollectiondate]) as DateTime,
      mcollectedamount:	map[TablesColumnFile.mcollectedamount] as double,
      mremark:    	map[TablesColumnFile.mremark] as String,
      mcashflow:map[TablesColumnFile.mcashflow] as String,
      mentrydate:(map[TablesColumnFile.mentrydate]=="null"||map[TablesColumnFile.mentrydate]==null)?null:DateTime.parse(map[TablesColumnFile.mentrydate]) as DateTime,
      mbatchcd:map[TablesColumnFile.mbatchcd] as String,
      merrormessage:map[TablesColumnFile.merrormessage] as String,
      msetno:map[TablesColumnFile.msetno] as int,
      mscrollno:map[TablesColumnFile.mscrollno] as int,
        missynctocoresys: map[TablesColumnFile.missynctocoresys] as int,
      moperationdate:map[TablesColumnFile.moperationdate] == "null" || map[TablesColumnFile.moperationdate] == null
          ? null
          : DateTime.parse(map[TablesColumnFile.moperationdate]) as DateTime,
    );
  }
}

File: Desktop/AsmaPravin/Masiudeen/Main_Transfer/Main_Transfer/eco_mfi/android/app/src/main/java/com/infrasofttech/eco_mfi/SavingsListBean.java
package com.infrasofttech.eco_mfi;


import java.time.LocalDateTime;

public class SavingsListBean {


    private int trefno = 0;
    private int mrefno;
    private int mlbrcode;
    private int mcusttrefno;
    private int mcustmrefno;
    private String mprdacctid;
    private String mmobno;
    private String mlongname;
    private String mcurcd;
    private String mprdcd;

    public int getTrefno() {
        return trefno;
    }

    public void setTrefno(int trefno) {
        this.trefno = trefno;
    }

    public int getMrefno() {
        return mrefno;
    }

    public void setMrefno(int mrefno) {
        this.mrefno = mrefno;
    }

    public int getMlbrcode() {
        return mlbrcode;
    }

    public void setMlbrcode(int mlbrcode) {
        this.mlbrcode = mlbrcode;
    }

    public int getMcusttrefno() {
        return mcusttrefno;
    }

    public void setMcusttrefno(int mcusttrefno) {
        this.mcusttrefno = mcusttrefno;
    }

    public int getMcustmrefno() {
        return mcustmrefno;
    }

    public void setMcustmrefno(int mcustmrefno) {
        this.mcustmrefno = mcustmrefno;
    }

    public String getMprdacctid() {
        return mprdacctid;
    }

    public void setMprdacctid(String mprdacctid) {
        this.mprdacctid = mprdacctid;
    }

    public String getMmobno() {
        return mmobno;
    }

    public void setMmobno(String mmobno) {
        this.mmobno = mmobno;
    }

    public String getMlongname() {
        return mlongname;
    }

    public void setMlongname(String mlongname) {
        this.mlongname = mlongname;
    }

    public String getMcurcd() {
        return mcurcd;
    }

    public void setMcurcd(String mcurcd) {
        this.mcurcd = mcurcd;
    }

    public String getMprdcd() {
        return mprdcd;
    }

    public void setMprdcd(String mprdcd) {
        this.mprdcd = mprdcd;
    }

    public LocalDateTime getMdateopen() {
        return mdateopen;
    }

    public void setMdateopen(LocalDateTime mdateopen) {
        this.mdateopen = mdateopen;
    }

    public LocalDateTime getMdateclosed() {
        return mdateclosed;
    }

    public void setMdateclosed(LocalDateTime mdateclosed) {
        this.mdateclosed = mdateclosed;
    }

    public int getMfreezetype() {
        return mfreezetype;
    }

    public void setMfreezetype(int mfreezetype) {
        this.mfreezetype = mfreezetype;
    }

    public int getMacctstat() {
        return macctstat;
    }

    public void setMacctstat(int macctstat) {
        this.macctstat = macctstat;
    }

    public int getMcustno() {
        return mcustno;
    }

    public void setMcustno(int mcustno) {
        this.mcustno = mcustno;
    }

    public double getMactTotbalfcy() {
        return mactTotbalfcy;
    }

    public void setMactTotbalfcy(double mactTotbalfcy) {
        this.mactTotbalfcy = mactTotbalfcy;
    }

    public double getMtotallienfcy() {
        return mtotallienfcy;
    }

    public void setMtotallienfcy(double mtotallienfcy) {
        this.mtotallienfcy = mtotallienfcy;
    }

    public int getMmoduletype() {
        return mmoduletype;
    }

    public void setMmoduletype(int mmoduletype) {
        this.mmoduletype = mmoduletype;
    }

    public String getMerrormessage() {
        return merrormessage;
    }

    public void setMerrormessage(String merrormessage) {
        this.merrormessage = merrormessage;
    }

    public int getMcenterid() {
        return mcenterid;
    }

    public void setMcenterid(int mcenterid) {
        this.mcenterid = mcenterid;
    }

    public int getMgroupcd() {
        return mgroupcd;
    }

    public void setMgroupcd(int mgroupcd) {
        this.mgroupcd = mgroupcd;
    }

    public String getMcrs() {
        return mcrs;
    }

    public void setMcrs(String mcrs) {
        this.mcrs = mcrs;
    }

    private LocalDateTime mdateopen;
    private LocalDateTime mdateclosed;
    private int mfreezetype;
    private int macctstat;
    private int mcustno;
    private double mactTotbalfcy = 0d;
    private double mtotallienfcy = 0d;
    private int mmoduletype;
    private String merrormessage = "";
    private int mcenterid;
    private int mgroupcd;
    private String mcrs;


}
